common/locales/en/merch.json

```diff
@@ -0,0 +1,10 @@
+{
+  "merch" : "Merchandise",
+
+  "merch-0-summary" : "Teespring is a platform that makes it easy for anyone to create and sell high-quality products people love, with no cost or risk.",
```

Comment:
Then in the jade file, you'd have something like:

```jade
- var merchants = []
- merchants.push({key: 'teespring', url: 'https://teespring.com/habitica-gryphon-t-shirt'})

each merchant, index in merchants
  if (index + 1) % 2 !== 0
    .row

  .col-md-6.col-sm-12
    .merch-block
      .merch-header: img(src="/merch/#{merchant.key}-logo.png")
      .merch-img-con: a(href=merchant.url target='_blank'): img.img-rendering-auto.merch-img(src="/merch/#{merchant.key}-sample.png")
      p=env.t('merch-#{merchant.key}-summary')
      a.btn.btn-habitica.merch-link(href=merchant.url target='_blank')=env.t('merch-#{merchant.key}-goto')
```

Then, if later on we wanted to add a separate product from teespring, we could just add a new key, `teespring2` and add

```jade
- merchants.push({key: 'teespring2', url: 'https://teespring.com/other-url'})
```

And the corresponding translation strings.